src/core/funcbloc.cpp

```diff
@@ -564,6 +541,9 @@ CFunctionBlock *CFunctionBlock::getFB(forte::core::TNameIdentifier::CIterator &p
     //only check for adpaters if it we have the last entry in the line
     retVal = getAdapter(*paNameListIt);
```

Comment:
I found an invalid nullptr access via `getAdapter` caused by an unititialized interface spec in `EMB_RES`, which manifests now because this is overriding `CFBContainer::getFB` and thus running this code in more situations. I have opened a PR https://github.com/azoitl/4diac-forte/pull/3 to address the problem, which also fixed the tests for me.

Reply:
This is an interesting issue. Because I tested in addition to the unit tests locally by deploying an app to 4diac FORTE.

Should we do some additional protection against nullptrs for interfacespecs? Should we turn it into an ref? Maybe in a separate PR.